packages/gas-fee-controller/package.json

```diff
@@ -39,11 +39,12 @@
     "@metamask/polling-controller": "^5.0.0",
     "@metamask/utils": "^8.3.0",
     "@types/uuid": "^8.3.0",
-    "ethereumjs-util": "^7.0.10",
+    "bn.js": "^5.2.1",
     "uuid": "^8.3.2"
   },
   "devDependencies": {
     "@metamask/auto-changelog": "^3.4.4",
+    "@types/bn.js": "^5.1.5",
```

Comment:
When I run `yarn build` I see this line in `packages/gas-fee-controller/dist/fetchBlockFeeHistory.d.ts`:

``` typescript
import BN from 'bn.js';
```

For this reason I think this needs to be in `dependencies` and not `devDependencies`, otherwise TypeScript consumers will need to add this package themselves.